src/lib/bootloader/sections.rb

```diff
@@ -22,12 +23,9 @@ def default
 
       return @default = "" if Yast::Stage.initial
 
-      # Execute.on_target can return nil if call failed. It shows users error popup, but bootloader
-      # can continue with empty default section
-      saved = Yast::Execute.on_target("/usr/bin/grub2-editenv", "list", stdout: :capture) || ""
-      saved_line = saved.lines.grep(/saved_entry=/).first
+      default_path = read_default
 
-      @default = saved_line ? saved_line[/saved_entry=(.*)$/, 1] : all.first
+      @default = default_path ? path_to_title(default_path) : all.first
```

Comment:
comment `attr :all` with `# titles of all sections`
to distinguish from `# paths of all sections` ?